test/cpp_api_parity/parity-tracker.md

```diff
@@ -97,6 +97,7 @@ torch::nn::TransformerEncoderLayer|Yes|No
 torch::nn::TransformerDecoderLayer|Yes|No
 torch::nn::Identity|Yes|No
 torch::nn::Linear|Yes|No
+torch::nn::Bias|Yes|No
```

Comment:
An entry should be added to the parity tracker for the functional form `F::bias` as well